netcore/tests/OPADotNet.Embedded.Tests/OPADotNet.Embedded.Tests.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

```suggestion
		<TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```

Since we need to be backwards compatible